msal_extensions/persistence.py

```diff
@@ -150,8 +153,13 @@ def save(self, content):
 
     def load(self):
         with self._Keychain() as locker:
-            return locker.get_generic_password(
-                self._service_name, self._account_name)
+            from .osx import KeychainError
```

Comment:
Sorry I was not always precise, but I mean to say [(at the very end of here)](https://github.com/AzureAD/microsoft-authentication-extensions-for-python/pull/65#pullrequestreview-458826870) to fix it in the token_cache layer.

And then even those new test cases would probably need to be moved to a new test_token_cache.py instead.

Reply:
Hey @rayluo , 
I understand your concern to add it in the token_cache layer but we can sync up offline on the complications involved in moving it there. Since this is a OS specific error, and token_cache layer is shared across OS, there will eventually be import errors for platforms that are not macOS as they wouldnt recognize the except statement and fail. I still think it should just be added in osx.py like it previously was. But, I do understand your concern so  adding it in persistence layer would be still better than token_cache.py.

Reply:
Let's move this import to before the `with ...` statement.